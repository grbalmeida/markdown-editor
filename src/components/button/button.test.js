'use strict'

import React from 'react'
import Button from './index'
import renderer from 'react-test-renderer'

it('Should Button to match snapshot', () => {
  const tree = renderer.create(
    <Button onClick={() => null}>Click me</Button>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

it('Should Button success to match snapshot', () => {
  const tree = renderer.create(
    <Button onClick={() => null} kind='success'>Success</Button>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

it('Should Button danger to match snapshot', () => {
  const tree = renderer.create(
    <Button onClick={() => null} kind='danger'>Danger</Button>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

