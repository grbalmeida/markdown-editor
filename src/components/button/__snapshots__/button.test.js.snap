// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Button danger to match snapshot 1`] = `
<button
  className="button -danger"
  onClick={[Function]}
>
  Danger
</button>
`;

exports[`Should Button success to match snapshot 1`] = `
<button
  className="button -success"
  onClick={[Function]}
>
  Success
</button>
`;

exports[`Should Button to match snapshot 1`] = `
<button
  className="button"
  onClick={[Function]}
>
  Click me
</button>
`;
